ClientApp/viking-vault/src/UserDashboard/ExchangeForm.tsx

```diff
@@ -228,8 +249,8 @@ class ExchangeForm extends React.Component<any, IExchangeFormState> {
         }
         else
         {
-            var amount = this.state.toExchangeAmount;
-            var otherParty = balance.toFixed(3) + " " + this.state.toCurrency;
+            var amount = this.state.exchangedAmount.toFixed(2);
+            let otherParty = `Exchanged ${this.state.toExchangeAmount} ${this.state.fromCurrency}`;
```

Comment:
What does otherParty stands for?

Reply:
There are two parties involved in a transaction: current user and the other party

Reply:
Why not sender and receiver then? It would make more sens.

Reply:
Because it can be: sender, receiver, company (that receives from automatic payments). So I don't this that Sender/Receiver/Company is a good column name

Reply:
Receiver works for both a company and a user.

Reply:
Same goes with sender.

Reply:
So I should name the column in the table Sender/Receiver?

Reply:
Yes, please.